soc/nxp/kinetis/Kconfig.soc

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2016 Intel Corporation
```

Comment:
This seems like a new file. If this is the case the copyright should be updated.